src/modules/hotel/hotel.controller.ts

```diff
@@ -0,0 +1,102 @@
+import { FastifyReply, FastifyRequest } from "fastify";
+import {
+    createHotel,
+    browseHotel,
+    showHotel,
+    updateHotel,
+    deleteHotel
+} from "./hotel.service";
+import {
+    CreateHotelInput,
+    UpdateHotelInput,
+    ShowHotelParams,
+    DeleteHotelParams
+} from "./hotel.schema";
+import { error_message } from "./hotel.errors";
+import { Hotel } from "@prisma/client";
+
+// In Seconds
+const cache_ttl = 1800;
+
+const cache_key_hotels = "allHotels";
+const cache_key_hotel = "hotel";
```

Comment:
All consts like this should be all caps.

```suggestion
const CACHE_TTL = 1800;

const CACHE_KEY_HOTELS = "allHotels";
const CACHE_KEY_HOTEL = "hotel";
```

Reply:
Changes added